benchmark/utils/formats.hpp

```diff
@@ -166,8 +176,9 @@ std::unique_ptr<MatrixType> read_matrix_from_data(
 const std::map<std::string, std::function<std::unique_ptr<gko::LinOp>(
                                 std::shared_ptr<const gko::Executor>,
                                 const gko::matrix_data<> &)>>
-    matrix_factory{
-        {"csr", READ_MATRIX(csr, std::make_shared<csr::automatical>())},
+    matrix_factory
+{
+    {"csr", READ_MATRIX(csr, std::make_shared<csr::automatical>())},
```

Comment:
weird indent ?

Reply:
clang-format seems to do different behavior between `#if` and `#ifdef`
If all begin with `#ifdef`, the format will be the same as before.
If some begin with `#if`, the format will be like here.
I use `#clang-format off` and `on` to format it now.
Does anyone have idea to handle `#if` and `#ifdef`?